threat-model/readme.md

```diff
@@ -216,20 +216,28 @@ TODO
 
 ### 1.10 TEE Manufacturers
 
-TEEs can provide "attestation" which verifies that the TEE is running in the expected state and running the expected code. These attestations are typically produced with an asymmetric key, where the private key is physically embedded into the TEE, and the public key is published via a system similar to a certificate authority.
+The TEE manufacturer is responsible for constructing the physical TEE hardware according to a specification and managing the associated public key infrastructure (PKI).
 
 #### 1.9.1 Assets
 
-1. Private keys embedded in TEEs and their corresponding public keys.
+1. Private keys embedded in TEEs. These mush be confidential and known only to the TEE. 
+2. List of public keys corresponding to valid attestation/identity private keys embedded in TEEs. This list must be correct in that it contains only the public keys corresponding to private keys embedded in the hardware of TEEs that were manufactured according to the desired specification. If this is not the case, then valid attestations may be provided by hardware that does have the expected properties or guarantees. It must also be public so that parties other than the manufacturer are able to recognize valid TEE hardware.
+3. TEE hardware
+4. Specification for TEE design
 
 #### 1.9.2 Capabilities
 
-1. If an attacker controls both the cloud provider and the TEE manufacturer, decrypt all data within the TEE.
+1. Construct TEEs
+2. Embed private attestation/identity keys in TEEs
+3. Maintain PKI for TEEs
```

Comment:
Given the framing above, this is more about moving public keys in or out of the set of those that are considered valid.  That a PKI is used for this is detail.

Reply:
Agreed, but doesn't calling it a PKI still work here? However it's implemented it will still need to map public keys to valid identities.

Reply:
My understanding is that "PKI" has very specific connotations for most people.

Reply:
changed to "Act as authority on which keys are considered valid TEE attestation/identity keys" in [f01ced2](https://github.com/patcg/docs-and-reports/pull/34/commits/f01ced284f47ba516cb8b6262a0eea42693572e1)